src/test-utils/test-server.ts

```
@@ -77,8 +87,20 @@ export const startApolloTesting = (): TestingContext => {
   })
   const { query, mutate } = createTestClient(server)
   return {
-    query,
-    mutate,
+    query: async (q) => {
+      const result = await query(q)
+      if (result.errors) {
+        throw result.errors
+      }
+      return result.data!!
+    },
+    mutate: async (m) => {
+      const result = await mutate(m)
+      if (result.errors) {
+        throw result.errors
+      }
+      return result.data!!
+    },
```

This change made errors show up as thrown instead of part of an `errors` object of the return value. Throwing them means they show up way easier in the tests